openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -20,16 +22,54 @@ def process(self, context):
 
         self.log.debug("Comment is `{}`".format(publish_comment))
 
+        families = set(
+            instance.data.get("kitsu_task")
+            for instance in context
+            if instance.data.get("publish")
+        )
```

Comment:
Please add comment to explain and I'd use the comprehension this way.
```suggestion
        families = {
            instance.data.get("kitsu_task")
            for instance in context
            if instance.data.get("publish")
        }
```